master/internal/configpolicy/task_config_policy.go

```diff
@@ -5,37 +5,22 @@ import (
 	"github.com/determined-ai/determined/master/pkg/schemas/expconf"
 )
 
-// ResourceConstraints are non-overridable resource constraints.
-// Submitted workloads that request resource quanities exceeding defined resource constraints in a
-// given scope are rejected.
-type ResourceConstraints struct {
-	MaxSlots *int `json:"max_slots"`
-}
-
-// Constraints are non-overridable workload constraints.
-// Submitted workloads whose config's respective field(s) exceed defined constraints within a given
-// scope are rejected.
-type Constraints struct {
-	ResourceConstraints *ResourceConstraints `json:"resources"`
-	PriorityLimit       *int                 `json:"priority_limit"`
-}
-
-// ExperimentConfigPolicy is the invariant config and constraints for an experiment.
+// ExperimentConfigPolicies is the invariant config and constraints for an experiment.
 // Submitted experiments whose config fields vary from the respective InvariantConfig fields set
 // within a given scope are silently overridden.
 // Submitted experiments whose constraint fields vary from the respective Constraint fields set
 // within a given scope are rejected.
-type ExperimentConfigPolicy struct {
-	InvariantConfig *expconf.ExperimentConfig `json:"invariant_config"`
-	Constraints     *Constraints              `json:"constraints"`
+type ExperimentConfigPolicies struct {
+	InvariantConfig *expconf.ExperimentConfig `json:"config"`
```

Comment:
Maybe I missed this discussion but what was the motivation behind changing this from `invariant_config` to `config`?

Reply:
This should indeed be `invariant_config`, great catch!

Reply:
Hm, ok I'm actually wondering, is it worth adding an extra select query to check this? Techncially, if the workspace ID is invalid, this is should be found in the API layer before the data gets passed to the db (I guess I think of this layer as solely for performing the requested DB operations and nothing else if that makes sense).
@kkunapuli any opinions here?